(#60) Welsh county names should not include #

hugovk opened the pull request:
Fixes #60.

Two commits, the first just to indent to make the second easier to check.

I also made some corrections to make the names match the official ones used by their councils:
 * Drop the definite article in "the Vale of Glamorgan" http://www.valeofglamorgan.gov.uk/en/index.aspx
 * Use the Welsh "Rhondda Cynon Taf" for English as well http://www.rctcbc.gov.uk/
 * Include the *to bach* (little hat) on the o for "Ynys Môn" https://www.ynysmon.gov.uk/

Questions:

 * Should these also include a `"wof:lang"`? 

  * If so, should it be `"wof:lang": [ "eng", "cym" ]` or `"wof:lang": [ "cym" ]`, as there's only a `name:cym*` and no `name:eng*`?

nvkelso commented:
@thisisaaronland to comment about splitting "eng#welsh" in the name field out into separate properties. 

1. Is `"name:cym_p" : "Sir Benfro"` the desired syntax to store the Welsh name variant? Should this be in the `wof`namespace?
2. Should the English name also be added to `"name:eng_p" : "Pembrokeshire"` (Hugo already proposes it be in `"wof:name" : "Pembrokeshire",`
3. How do we indicate both name variants are common local usage (versus an exonym in that language).

thisisaaronland commented:
A few comments:

I would propose `wel` instead of `cym`. The former is the bibliographic alpha-3 code as opposed to the terminologic code, per ISO-639-2:

https://github.com/whosonfirst/py-mapzen-whosonfirst-languages/blob/master/data/ISO-639-2_utf-8-2015-09-21.txt#L466

http://www.loc.gov/standards/iso639-2/ascii_8bits.html

In the case of Wales, including both the English and Welsh pointers in `wof:lang` seems reasonable given the status of the language:

https://en.wikipedia.org/wiki/Welsh_language#Official_status

I need to backfill `wof:lang` from the Geoplanet dumps (oops) which will only contain a single language and not account for places like Wales, Quebec, Catalonia, Belguim.... I mean, really the list just goes on and on and on. But one thing at a time :D

I think "common usage" should probably be the criteria for inclusion in the `wof:lang` (rather than strict administrative interpretation).

hugovk commented:
PR updated:

 * Replaced `cym` with `wel`
 * Added `"wof:lang": [ "eng", "wel" ],`
 * Split `"wof:name" : "Conwy#Conwy",` which I missed first time
